terraform/terraform.tfstate

```diff
@@ -10,6 +10,45 @@
             ],
             "outputs": {},
             "resources": {
+                "aws_cloudwatch_metric_alarm.failure": {
+                    "type": "aws_cloudwatch_metric_alarm",
+                    "depends_on": [
+                        "module.email"
+                    ],
+                    "primary": {
+                        "id": "inclucivics_ingest_failure",
+                        "attributes": {
+                            "actions_enabled": "true",
+                            "alarm_actions.#": "1",
+                            "alarm_actions.3866892562": "arn:aws:sns:us-east-1:605366678235:inclucivics-ingest-email-EmailSNSTopic-10PQOZULTIXGP",
```

Comment:
We've indicated our account number with this ARN, but still no passwords. 👍

Reply:
I see where this is getting a little risky.